bin/mail/2017-05-17-end-of-year/send-final-real-mails

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+../send-to-mailing-list 2017-05-17-endofyear-k5pd ./email_1_csf_pd.csv
+../send-to-mailing-list 2017-05-17-endofyear-k5nopd ./email_2_csf_no_pd-remainder.csv
```

Comment:
Oh, I thought the winner was only relevant to that one template and we were sending the other 5 at the same time as the A/B test for `k5nopd`, then the winner later. In that case, this is fine.